interval.md

```diff
@@ -10,16 +10,25 @@ The `INTERVAL` [data type](data-types.html) stores a value that represents a spa
 
 ## Formats
 
-When inserting into an `INTERVAL` column, use one of the following formats:
+Constant values of type `INTERVAL` must be expressed using an
```

Comment:
Let's change the section title to `Syntax` and rephrase the first sentence to match the others:

```
Values of type `INTERVAL` must be expressed as [interpreted literals](sql-constants.html#interpreted-literals) or string literals [annotated with](sql-expressions.html#explicitly-typed-expressions) type `INTERVAL` or [coerced to](sql-expressions.html#explicit-type-coercions) type `INTERVAL`.
```